add gaps in funding (#5)
adamjonas/adamjonas.github.io: _posts/2022-06-03-gaps-in-bitcoin-dev-funding.markdown

---
layout: post
title:  "The gaps in bitcoin open-source funding"
date:   2022-06-03 09:57:39 -0500
categories: bitcoin funding grants
---

> A few people reached out about my post on my [Guide for Grant Seekers](/bitcoin/funding/grants/grants-bitcoin-open-source/) with some critical feedback about promoting work for free culture. To clarify, this guide was not meant to be an ideal vision for the future, it was meant to reflect the present and help grant seekers achieve their goals.

Reviewer: ```suggestion
A few people reached out about my post on my [Guide for Grant Seekers](/bitcoin/funding/grants/grants-bitcoin-open-source/) with some critical feedback about promoting work-for-free culture. To clarify, this guide was not meant to be an ideal vision for the future; it was meant to reflect the present and help grant seekers achieve their goals.
```

Reviewer: I would hyphenate "work for free" because it is the name of the culture we are referring to, it can stay as is in the other places it's used her.

> At the end of the post, I mentioned that this doesn't serve a lot of different groups. In fact, I think it serves a limited population which is why we see such geographic density of bitcoin FOSS devs. People that cannot do work for free over a long period aren't going to be able to make significant progress on becoming a FOSS dev quickly. Working on an all-consuming start-up, raising a family, or living in a place with limited electricity/internet will put grant seekers at a significant disadvantage. Because the proof of work is lower for other projects, this also means that, by default, they are more inclusive. I am biased because free work is such a useful screen. It's almost unforgeable proof of intrinsic motivation.

Reviewer: At the end of this paragraph, you talk about "Proof-of-work is lower for other projects, this also means that, by default, they are more inclusive", can we be more specific as to what is being referred to here? 

Obviously it's referring to other "crypto" projects; as long as the adamjonas blog stays focused on this subject, then this could stay as is :)

Reviewer: I think Bitcoin is very inclusive, and its inclusiveness is a function of how easily one can join the "community". There's a million bitcoin dev meetups, devs will reach out to you if you're a promising new contributor, no barrier of entry to submit a review or code change (outside of defaults like education, computer, internet), most devs are very friendly. 

How inclusive it is at paying people to work on it is a different story and, to me, is a function of a social structure on top of the bitcoin dev community where philanthropic money is distributed to individuals. And philanthropic money is less accessible than VC scam funds.

Author: lower standards are by definition more inclusive.

> At the end of the post, I mentioned that this doesn't serve a lot of different groups. In fact, I think it serves a limited population which is why we see such geographic density of bitcoin FOSS devs. People that cannot do work for free over a long period aren't going to be able to make significant progress on becoming a FOSS dev quickly. Working on an all-consuming start-up, raising a family, or living in a place with limited electricity/internet will put grant seekers at a significant disadvantage. Because the proof of work is lower for other projects, this also means that, by default, they are more inclusive. I am biased because free work is such a useful screen. It's almost unforgeable proof of intrinsic motivation.

Reviewer: I think this sounds better
```suggestion
At the end of the post, I mentioned that this doesn't serve a lot of different groups. In fact, I think it serves a limited population which is why we see such geographic density of bitcoin FOSS devs. Those who cannot do work for free over a long period aren't going to be able to make significant progress on becoming a FOSS dev quickly. Working on an all-consuming start-up, raising a family, or living in a place with limited electricity/internet will put grant seekers at a significant disadvantage. Because the proof of work is lower for other projects, this also means that, by default, they are more inclusive. I am biased because free work is such a useful screen. It's almost unforgeable proof of intrinsic motivation.
```

## A funding bridge

> The big gap I see that isn't currently filled in the funding landscape is for people who can't quit their job or can only afford to work nights and weekends because of other commitments. Sure we see part-time grants, but building up one's credibility to get to that point takes investment. Funders aren't really taking a chance on people. The [residency](https://residency.chaincode.com/) filled this gap for some in the past. It was an almost-guarantee of finding FOSS funding. The residencies were never meant to scale and they are in hibernation anyways. Instead, we need a pot of cash to fund prospects. Let's call them bridge grants.

Reviewer: ```suggestion
The big gap I see that isn't currently filled in the funding landscape is for people who can't quit their job or can only afford to work nights and weekends because of other commitments. Sure we see part-time grants, but building up one's credibility to get to that point takes investment. Funders aren't really taking a chance on people. The [residency](https://residency.chaincode.com/) filled this gap for some in the past. It was an almost-guarantee of finding FOSS funding. The residencies were never meant to scale, and they are in hibernation anyways. Instead, we need a pot of cash to fund prospects. Let's call them bridge grants.
```

Bridge grants are short-term grants, somewhere between 3 and 6 months, to wean a prospect off their job and give them a runway to show what they can do. They won't get rich from this funding, but it will be enough to avoid starvation and should inspire them to spend their time wisely establishing a track record that will attract a full-fledged grant.

## A manager

No strings attached funding provides the freedom to do meaningful work. In an industrialist society that no longer offers much respect to the individual worker, it reflects the trust of the grantee to choose the work that needs to be done. It's a thing of beauty.

> But, without strings, grantees don't go through the same growth of learning how to be professional -- AKA, how to show up when they don't feel like it. The kind of pain that comes with showing up to a bad job day after day because you made a promise to be there. No strings attached open-source funding doesn't help develop young people to be critical of management or understand when they need to quit a job. When a manager isn't parceling out tasks or providing (even poorly done) performance reviews, I believe it is harder for a green dev to get the feedback they need to grow as a developer and generally be consistently productive. Because FOSS doesn't signal whether you are performing well, I've seen people become demotivated and discouraged without a feedback structure that works (see wildly successful talent recruitment and retention of large tech companies).

Reviewer: ```suggestion
But, without strings, grantees don't go through the same growth of learning how to be professional -- AKA, how to show up when they don't feel like it. The kind of pain that comes with showing up to a lousy job day after day because you made a promise to be there. No strings attached open-source funding doesn't help develop young people to be critical of management or understand when they need to quit a job. When a manager isn't parceling out tasks or providing (even poorly done) performance reviews, I believe it is harder for a green dev to get the feedback they need to grow as a developer and generally be consistently productive. Because FOSS doesn't signal whether you are performing well, I've seen people become demotivated and discouraged without a feedback structure that works (see wildly successful talent recruitment and retention of large tech companies).
```

## A career

> Bitcoin open-source development isn't yet a career. Even in the US, where taking the road less traveled is culturally celebrated, health care is tied to employment. There is motivational value in career progression. While I don't believe in titles (see current descriptive title - Head of Special Projects), demonstrating career progression is helpful to mitigate the criticisms of friends and family who are already skeptical of the open-source path.

Reviewer: I think the second sentence here needs to be developed more. And in general this section should be expanded upon.

Reviewer: Career: "an occupation undertaken for a significant period of a person's life and with opportunities for progress."

There's opportunities to progress bitcoin :D

Author: i disagree

## Something else

> We need a prospect league that is a scaled-down version of the current grant system with a greater margin of error and more structure. This development program provides the financial safety and modeling to invite more of those that otherwise couldn't make the jump to bitcoin open-source development. This is the strategy behind programs like Summer of Bitcoin and Qala. Get a foothold. Find promising people to invest more in. We have to do it on a larger scale. FOSS development [can be hard](/bitcoin/managing-yourself-in-open-source/) and so we can either say that devs without 5+ years of wading through industry jobs are the only people that should attempt it, or we can create better scaffolding and structure for those that are greener.

Reviewer: This seems like it would combine nicely with the content under "A Funding Bridge"

Author: I tried to combine but then I didn't have a conclusion. So I mostly left them separated.

> We need a prospect league that is a scaled-down version of the current grant system with a greater margin of error and more structure. This development program provides the financial safety and modeling to invite more of those that otherwise couldn't make the jump to bitcoin open-source development. This is the strategy behind programs like Summer of Bitcoin and Qala. Get a foothold. Find promising people to invest more in. We have to do it on a larger scale. FOSS development [can be hard](/bitcoin/managing-yourself-in-open-source/) and so we can either say that devs without 5+ years of wading through industry jobs are the only people that should attempt it, or we can create better scaffolding and structure for those that are greener.

Reviewer: ```suggestion
We need a prospect league that is a scaled-down version of the current grant system with a greater margin of error and more structure. This development program provides the financial safety and modeling to invite more of those that otherwise couldn't make the jump to bitcoin open-source development. This is the strategy behind programs like Summer of Bitcoin and Qala. Get a foothold. Find promising people to invest more in. We have to do it on a larger scale. FOSS development [can be hard](/bitcoin/managing-yourself-in-open-source/). We can either say that devs without 5+ years of wading through industry jobs are the only people who should attempt it, or we can create better scaffolding and structure for those that are greener.
```
